src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -767,7 +767,13 @@ def perform_destroy(self, instance):
     ),
     create=extend_schema(
         summary="Maak een ZAAKEIGENSCHAP aan.",
-        description="Maak een ZAAKEIGENSCHAP aan.",
+        description=(
+            "Maak een ZAAKEIGENSCHAP aan.\n"
+            "\n"
+            "**Er wordt gevalideerd op**\n"
+            "\n"
+            "- **EXPERIMENTEEL** `waarde` moet worden gevalideerd tegen `eigenschap.specificatie`."
```

Comment:
Small nitpick but this description might make it seem like the validation would always run (which is the case if the option is enabled through the added env variable).
```suggestion
            "- **EXPERIMENTEEL** `waarde` moet worden gevalideerd tegen `eigenschap.specificatie` indien `ZAAK_EIGENSCHAP_WAARDE_VALIDATION` op `True` staat."
```